City-of-Helsinki/drupal-helfi-platform-config
# UHF-8386 CKEditor5 quote

# [UHF-8386](https://helsinkisolutionoffice.atlassian.net/browse/UHF-8386)


❗ This PR should be merged into `ckeditor5` branch - 🛑 not `main`

## What was done


* Added quote plugin for ckeditor5.
* Added editor and filter format settings to configurations.
* Fixed the translation tools for the CKE5

## How to install
* Make sure your instance is up and running on latest dev branch.
    * `git pull origin dev`
    * `make fresh`
* Update the Helfi Platform config
    * `composer require drupal/helfi_platform_config:dev-UHF-8386_ckeditor5_quote`
* Run `make drush-updb drush-cr`
* Import translations
    * Run `make shell` and in shell: `drush locale:check; drush locale:update; drush cr`  

## How to test


* [x] Go to edit some content via CKEditor and check that you can select a portion of text and make a quote out of it. 
* [x] Save the content and check that the quote rendered with the correct styles

Check the images below. 👇 
![image](https://github.com/City-of-Helsinki/drupal-helfi-platform-config/assets/1712902/05946630-431f-4d89-b5c8-82bc7e4360d5)
![image](https://github.com/City-of-Helsinki/drupal-helfi-platform-config/assets/1712902/ae0e451c-465a-445e-8978-5753e76ae671)
![image](https://github.com/City-of-Helsinki/drupal-helfi-platform-config/assets/1712902/79e27195-929d-42ee-8e33-299649ab15b7)

[UHF-8386]: https://helsinkisolutionoffice.atlassian.net/browse/UHF-8386?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
